docs/source/conf.py

```diff
@@ -253,3 +257,22 @@ def process_readme():
 
 
 process_readme()
+
+# Auto-generate documentation pages
+current_dir = os.path.abspath(os.path.dirname(__file__))
+module = os.path.join(current_dir, "..", "..", "sentinelhub")
+
+APIDOC_EXCLUDE = [os.path.join(module, "commands.py")]
```

Comment:
I would also put `sentinelhub.aws.commands` module to this list.

Reply:
done